homie/examples/sensor.rs

```diff
@@ -28,10 +28,18 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync>> {
                     "temperature",
                     "Temperature",
                     Datatype::Float,
+                    false,
                     Some("ºC"),
                     None,
                 ),
-                Property::new("humidity", "Humidity", Datatype::Integer, Some("%"), None),
+                Property::new(
+                    "humidity",
+                    "Humidity",
+                    Datatype::Integer,
+                    false,
```

Comment:
Could we have something like this:
```
enum ... {
    ReadOnly,
    ReadWrite(callback),
}
```
?

Reply:
Hrm. `settable` true/false does fit better with the language used in homie. Maybe leave it as it is?

Reply:
Ooh, I quite like that if we do end up moving the callback here.